spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -137,16 +136,18 @@ public class Student {
 
 In this example, the `name` field is annotated with `@NotBlank`, indicating that it must not be blank. The `age` field is annotated with both `@Positive` and `@Min(18)`, ensuring it is a positive integer greater than or equal to 18.
 
-The `School` repository interface, shown below, utilizes the validation rules defined in the `Student` entity:
+The `School` repository interface, shown below, uses the `jakarta.validation.Valid` annotation to cause the constraints from the `Student` entity to be validated during the `save` operation, whereas the validation constraints are not applied to the `Student` entities returned as a result of the `findByAgeLessThanEqual` operation because the `findByAgeLessThanEqual` method does not include a `jakarta.validation.Valid` annotation that applies to the return value.
 
 [source,java]
 ----
 @Repository
-public interface School extends PageableRepository<Student, String> {
+public interface School extends DataRepository<Student, String> {
+    void save(@Valid Student s);
```

Comment:
```suggestion
````

Reply:
If this makes sense the semantic, I would remove this line.

Reply:
See explanation in response to previous comment.

Reply:
Thanks.